modules/ROOT/pages/appendix/gql-conformance/unsupported-mandatory.adoc

```diff
@@ -46,9 +46,7 @@ The only way to guarantee row order in Neo4j is to use xref:clauses/order-by[`OR
 
 | 15.1 & 15.2
 | `CALL` procedure statement & inline procedure `CALL`
-| * GQL defines an `OPTIONAL CALL` statement, enabling optional procedure and subquery calling.
-* GQL implicitly imports variables.
```

Comment:
`* GQL implicitly imports variables.` should still be preserved as this is GP02 which will be implemented at a later time.

Reply:
This will be rephrased in a different GQL specific PR - it will read like this:

"GQL either imports variables implicitly, or explicitly using a variable scope clause.
In Cypher, `CALL` subqueries require an explicit importing `WITH` clause."

And, once this PR ready, the second sentence in that passage will say something like "Cypher requires explicitly importing variables"